community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup.py

```diff
@@ -435,6 +435,9 @@ def setup_controller():
 
     if not lkp.cfg.cloudsql_secret:
         configure_mysql(lkp)
+    
+    if lkp.has_flex():
```

Comment:
this is not an invariant. `has_flex` can change it's value in lifetime of the cluster.

Reply:
Good point, although its usage is in a function that should only run once. Are you suggestion we move it outside of Lookup?

Reply:
resolved offline, it was about post cluster additions